sunpy/timeseries/timeseriesbase.py

```diff
@@ -334,6 +335,10 @@ def concatenate(self, otherts, **kwargs):
         at once.
         """
 
+        # check to see if nothing needs to be done
+        if self == otherts:
```

Comment:
how is this going to be computed? is `__eq__` defined? would this bet better as `is`??